shared/macros/10-kubernetes.jinja

```diff
@@ -219,9 +231,11 @@ spec:
 
 {{#
   High level macro to generate Kubernetes remediation to set the ssh daemon configuration file.
+  Parameters:
+    ocp_version_range: Optional OpenShift version range (default: '<4.13.0')
 #}}
-{{%- macro kubernetes_sshd_set() -%}}
-{{{ kubernetes_machine_config_file(path='/etc/ssh/sshd_config', file_permissions_mode='0600', source=sshd_config_source()) }}}
+{{%- macro kubernetes_sshd_set(ocp_version_range='<=4.13.0') -%}}
```

Comment:
Shouldn't this be <4.13.0 or <=4.12.0?

Reply:
Yes, documentation is correct.

Just to be more verbose:
`<=4.12.0` doesn't work as it doesn't into account bigger patch version like `4.12.1`.
And `4.12` is not allowed, it needs to have major, minor and patch numbers.